### DISCUSSION TITLE
Splice9: a pure vim9script port of splice

### DISCUSSION THREAD

**Discussion by errael** [errael/splice9#15] [Announcements]
To whom it may concern 🤓

Introducing [splice9](https://github.com/errael/splice9)

Enhancements are primarily an active `Heads Up Display`; most of the "buttons" execute if clicked and there are additional state indicators; see [Dynamic HUD](https://github.com/errael/splice9#dynamic-hud) for a description of the new `HUD` features.

Install splice9 (it uses your existing configuration) and enable it for your VCS. If splice9 finds `g:splice_config`, that dictionary has the configuration information; so splice9 and splice can both be installed and have different configurations.

There are new configuration options, including
- each command can be explicitly mapped
- highlights are configurable
- option to use `Alt`/`Meta` instead of a prefix

[Download splice9 v0.9 from here](https://github.com/errael/splice9/releases).

The recent vim9.1 release is required to use splice9.

**Comment by dlamotte** [comment]
Just wanted to say thanks for the update and the ping. Started using it today and it felt snappy and worked well.
